### ISSUE TITLE
accumulo-cluster does not start a CompactionCoordinator on different host

### ISSUE THREAD

**Issue by ddanielr** [apache/accumulo#4597]
**Describe the bug**
The accumulo-cluster script does not start a compaction-coordinator process if the defined host is different than the local manager node.
 
 SSHing to the expected node, an `accumulo-cluster start-here` command also failed to start the coordinator. 
 
**Versions (OS, Maven, Java, and others, as appropriate):**
 - Affected version(s) of this project: [e.g. 1.10.0] 2.1.x
 - OS: [e.g. CentOS 7.5] centos 7
 - Others:

**To Reproduce**
1. Modify the cluster.yaml to have the compaction-coordinator service defined on a host other than localhost and the hostname used for the gc, monitor, and manager.
1. Run an `accumulo-cluster start` command
1. SSH to the expected host and ensure no process is running. 
1. Make sure the cluster.yaml is updated on that host then run `./accumulo-cluster start-here`. 
1. Verify that the process did not attempt to start by looking at the log files or pid files in accumulo's run directory.
1. Finally, run a `accumulo-service compaction-coordinator start` operation and verify the process attempted to start and is running.

**Expected behavior**
The cluster script is able to start and stop the compaction-coordinator no matter what host it is running on.

**Comment by dlmarion** [comment]
I was unable to replicate this. I modified `accumulo-cluster` locally to emit debug output and so that it would always take the `ssh` path. I set the hosts in cluster.yaml to the public IP address of my dev box, but set my manager address to `localhost`. I saw the ssh command come across and compaction-coordinator logs appeared in the logs directory (Accumulo wouldn't start because I had ZK and Hadoop down).
